src/list.json

```diff
@@ -197,7 +197,18 @@
         "link": "https://blog.meilisearch.com/contribute-hacktoberfest-2022/"
       }
     ],
-    "N": [],
+    "N": [
+      {
+        "organization": "Novu",
+        "org_url" : "https://novu.co/",
+        "tags" :  [
+          "Tshirts"
+        ],
```

Comment:
Then Please Re-style it via pasting this command in the Repo Directory.
`git checkout <your branch> `
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2338620/patch | git am`
`git push `